AirCasting/SearchAndFollow/MapSessionsDownloader/API/SessionsForLocationDownloader.swift

```diff
@@ -34,9 +34,9 @@ class SessionsForLocationDownloaderDefault: SessionsForLocationDownloader {
                                        north: geoSquare.north,
                                        limit: 100,
                                        offset: 0,
-                                       sensorName: "openaq-pm2.5",
-                                       measurementType: MapDownloaderMeasurementType.particulateMatter.name,
-                                       unitSymbol: MapDownloaderUnitSymbol.uqm3.name)
+                                       sensorName: sensor.apiName,
+                                       measurementType: parameter.apiName,
+                                       unitSymbol: (parameter == .particulateMatter) ? MapDownloaderUnitSymbol.uqm3.name : MapDownloaderUnitSymbol.ppb.name)
```

Comment:
I know it's probably temporary here, but it has a flow I need to point out:
It we add a third unit symbol to `MapDownloaderMeasurementType` this code will break. We want it to not break :) It'd go with an extension